src/main/java/core/basesyntax/model/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.model;
+
+public class Bulldozer extends Machine {
+

```

Comment:
let's remove this empty line

Reply:
If I remove the space between package and public, mvn throws an error